.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: housekeeping
+on: [push, pull_request]
```

Comment:
The triggers should be more excplicit, not sure what the defaults are on e.g. `pull_request`. You can do something like
```yaml
on:
  push:
    branches:
    - '**'
  pull_request:
    types:
      - opened
      - synchronize
      - reopened  
```
And in the future if you wan't to exclude `main` branch from this workflow and run some deployment or something else there, you can add `- '!main'` under branches.

Reply:
Added this in 1cbf73c.